syz-ci/jobs.go

```diff
@@ -69,17 +66,18 @@ loop:
 		default:
 		}
 		select {
-		case <-ticker.C:
+		case <-jobTicker.C:
 			if len(kernelBuildSem) != 0 {
 				// If normal kernel build is in progress (usually on start), don't query jobs.
 				// Otherwise we claim a job, but can't start it for a while.
 				continue loop
 			}
 			jp.pollJobs()
-			if time.Since(lastCommitPoll) > commitPollPeriod {
-				jp.pollCommits()
-				lastCommitPoll = time.Now()
+		case <-commitTicker.C:
+			if len(kernelBuildSem) != 0 {
```

Comment:
This is not needed. pollCommits does not take kernelBuildSem.
kernelBuildSem can be taken for prolonged periods of time, and we poll once per hour.
With this check we can occasionally miss a number of hours.